nautobot/extras/models/mixins.py

```diff
@@ -18,3 +25,51 @@ def dynamic_groups(self):
             self._dynamic_group_queryset = queryset
 
         return self._dynamic_group_queryset
+
+
+# 2.0 TODO: Remove after v2 since we will no longer care about backwards-incompatibility.
```

Comment:
What would the 2.0 change/fix be here? Removing the mixin? What would the impact of that be?

Reply:
TODO: add these notes to the comment.